collector/fixtures/e2e-64k-page-output.txt

```diff
@@ -1,55 +1,59 @@
-# HELP go_gc_duration_seconds A summary of the pause duration of garbage collection cycles.
+# HELP go_gc_duration_seconds A summary of the wall-time pause (stop-the-world) duration in garbage collection cycles.
```

Comment:
are the fixture updates related to the slog transition? :thinking:

Reply:
No, this is related to the Go compiler version bump.